src/flb_input.c

```diff
@@ -1255,6 +1255,12 @@ int flb_input_init_all(struct flb_config *config)
             flb_input_instance_destroy(ins);
             return -1;
         }
+
```

Comment:
Move this block to the end of `flb_input_instance_init`.

Reply:
Sure, this should be done in https://github.com/fluent/fluent-bit/pull/7396/commits/35db25971716b2d6b0d5e4deaa8f51bc706548df.